examples/pytorch/diffpool/train.py

```diff
@@ -123,7 +123,7 @@ def graph_classify_task(prog_args):
     perform graph classification task
     '''
 
-    dataset = tu.TUDataset(name=prog_args.dataset)
+    dataset = tu.LegacyTUDataset(name=prog_args.dataset)
```

Comment:
Do the `LegacyTUDataset` and `TUDataset` have the same APIs?

Reply:
The processing code are quite different. In the new `TUDataset`, I used only used raw data without adding one-hot label or constant as node features.

Reply:
hope QM9 results will be online soon